### DISCUSSION TITLE
Remote server configuration are valid but still using test server

### DISCUSSION THREAD

**Discussion by ammar-bay** [saran-surya/email_auth_node#33] [Q&A]
Hey, when I start my app it says that my remote server configurations are valid, but when I send OTP it says that remote server is unavailable and uses test server instead, can you please guide me a little here? I am attaching both SS.
When app starts
![image](https://user-images.githubusercontent.com/93938575/170789673-44e4f80b-084a-42c8-8218-36ce82a02479.png)
When sending OTP
![image](https://user-images.githubusercontent.com/93938575/170789967-cef6f552-c452-47fc-a216-827e814c4577.png)

**Comment by saran-surya** [comment]
Hi @ammar-bay , 

- Post in your remote server configuration file
- Post in your heroku configuration
    - You can follow the steps https://saran-surya.github.io/email-auth-node/

**Reply by ammar-bay** [reply]
This is my remote server configuration file
![Screenshot 2022-05-28 102821](https://user-images.githubusercontent.com/93938575/170811550-89d69815-b997-48e4-ab62-481933acbe76.png)
I have followed all the steps mentioned here: https://saran-surya.github.io/email-auth-node/

**Reply by ammar-bay** [reply]
This is my server page:
![image](https://user-images.githubusercontent.com/93938575/170811653-98c5f2cc-b6f6-465e-ad90-93332de54881.png)

**Reply by ammar-bay** [reply]
These are my config vars on heroku:
![image](https://user-images.githubusercontent.com/93938575/170816030-d83897c7-a8f8-4a28-a579-90ab711915fd.png)

**Reply by saran-surya** [reply]
Your server is working, Can you check and update regarding the issue ?

**Comment by saran-surya** [comment]
Seems like you have added the build configuration, so kindly re-deploy the
server. And check again. If it fails again do post the heroku logs.

**Comment by saran-surya** [comment]
Marking as done, due to inactivity
